ee/hogai/graph/query_planner/nodes.py

```diff
@@ -54,7 +55,9 @@
 )
 
 
-class QueryPlannerNode(AssistantNode):
+class QueryPlannerNode(TaxonomyReasoningNodeMixin, AssistantNode):
+    REASONING_MESSAGE = "Picking relevant events and properties"
```

Comment:
As we mix `TaxonomyReasoningNodeMixin` now - it seems to use `Action.objects.aget`, the N+1 thing I commented before? Or it will not be used frequently?